src/rpm/package.rs

```diff
@@ -590,6 +590,55 @@ impl RPMPackageMetadata {
         )
     }
 
+    /// An RPM package is comprised of several segments - the Lead, Signature Header, Header, and Payload.
+    /// This function returns the computed (byte) boundaries between those segments in the package file
+    /// as if it were written out on-disk.
+    ///
+    /// ```
+    /// # use rpm::RPMPackage;
+    /// # let package = RPMPackage::open("test_assets/389-ds-base-devel-1.3.8.4-15.el7.x86_64.rpm").unwrap();
+    /// let (sig_header_start, header_start, payload_start) = package.metadata.get_package_segment_boundaries();
+    /// let lead = 0..sig_header_start;
+    /// let sig_header = sig_header_start..header_start;
+    /// let header = header_start..payload_start;
+    /// let payload = payload_start..;
+    /// ```
+    pub fn get_package_segment_boundaries(&self) -> (u64, u64, u64) {
```

Comment:
```suggestion
    pub fn get_package_segment_boundaries(&self) -> (u64, u64, u64) {
```

Please define a return struct type.